src/components/Organisation/Banner/hooks.ts

```diff
@@ -78,4 +79,63 @@ const useUpdateOrganisation = (organisationId: number) => {
   return updateOrgSocials;
 };
 
-export { useUpdateOrgSocials, useUpdateOrganisation };
+const useUpdateOrgPic = (organisationId: number) => {
+  const queryClient = useQueryClient();
+  const accessToken = getAccessToken();
+
+  const updateOrgSocials = useMutation(
```

Comment:
Can you follow the structure present in docs for mutation otherwise the error won't go through.